src/types.ts

```diff
@@ -183,20 +183,10 @@ export type MessageDefinitionField = {
   defaultValue?: DefaultValue;
 
   /**
-   * Set to true if this field is marked as deprecated in the message definition.
+   * Indicates whether this field is marked as deprecated in its message definition.
```

Comment:
This seems redundant, as the whole package is about message definitions :)
```suggestion
   * Indicates whether this field is marked as deprecated.
```